client/src/app/core/repositories/event-management/meeting-repository.service.ts

```diff
@@ -1,18 +1,30 @@
 import { Injectable } from '@angular/core';
 
+import { DEFAULT_FIELDSET, Fieldsets } from 'app/core/core-services/model-request-builder.service';
 import { Meeting } from 'app/shared/models/event-management/meeting';
 import { ViewMeeting } from 'app/site/event-management/models/view-meeting';
+import { MeetingSettingsService } from 'app/site/meeting-settings/meeting-settings.service';
```

Comment:
Please do not import from site into core

Reply:
What do you suggest? I think the current location of the meeting-settings.service suits  it best, it's not a core service and neither a repository, so I don't know where in core it could be placed...